.gitignore

```diff
@@ -2,3 +2,4 @@
 /node_modules/
 /coverage
 tmp
+.idea
```

Comment:
A bit nit-picky, but can you please remove this? We use global ignores for our own envs (like IDE of choice), leaving the project's ignore for files specifically relevant to the project.

Reply:
Done.

I avoid global ignores myself, because IIRC npm does not respect the global `.gitignore` during `npm publish`.